scio-test/src/main/scala/com/spotify/scio/testing/ApproxPipelineSpec.scala

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
This adds a couple of helper methods for testing with error rates, there are other ways to do this, 
- add these methods as matchers so we can call `actual shouldBeInRage (expected, errorRate)` 
- Add these as methods of implicits wrapper

I find adding these methods in an implicit class is a bit uglier and adding these in SCollectionMatcher also will pollute it, as these are specific to approximate algorithms. IMO, start with the current approach and later change it once we have a lot of approximation tests would be a good option. But let me know what you all think.

Reply:
I think I'm leaning towards having these as matches; that makes this consistent with the rest.

in regards to:

> adding these in SCollectionMatcher 

We can supply them as we supply `CoderAssertions`

https://github.com/spotify/scio/tree/master/scio-test/src/main/scala/com/spotify/scio/testing/CoderAssertions.scala